src/status_im/ui/screens/screens.cljs

```diff
@@ -51,7 +51,7 @@
     [status-im.ui.screens.profile.contact.views :as contact]
     [status-im.ui.screens.profile.group-chat.views :as profile.group-chat]
     [status-im.ui.screens.profile.seed.views :as profile.seed]
-    [status-im.ui.screens.profile.user.views :as profile.user]
+    [status-im2.contexts.profile.profiles.views-v2 :as profile.user]
```

Comment:
it would be preferable if you register the new screen in `status-im2.navigation.screens` namespace instead 👍